src/librustc_metadata/index.rs

```diff
@@ -111,13 +109,10 @@ impl Index {
         let pos = buf.position();
 
         // First we write the length of the lower range ...
-        buf.emit_raw_bytes(&(self.positions[0].len() as u32 / 4).to_le_bytes());
-        // ... then the values in the lower range ...
-        buf.emit_raw_bytes(&self.positions[0]);
-        // ... then the values in the higher range.
-        buf.emit_raw_bytes(&self.positions[1]);
-        LazySeq::with_position_and_length(pos as usize,
-            (self.positions[0].len() + self.positions[1].len()) / 4 + 1)
+        buf.emit_raw_bytes(&(self.positions.len() as u32 / 4).to_le_bytes());
```

Comment:
This is unnecessary, since it indicated the split between lo/hi, which doesn't exist anymore.